### DISCUSSION TITLE
💡 Skip login page with only 1 OIDC provider and no password auth

### DISCUSSION THREAD

**Discussion by alex3305** [henrygd/beszel#21] [Ideas]
When password auth is disabled and only a single OIDC provider is enabled, the login screen becomes a bit redundant:
![image](https://github.com/user-attachments/assets/a6c53d0f-6170-4f62-b443-9cddf2b401fa)

Perhaps this page could be skipped altogether in this configuration?

**Comment by henrygd** [comment]
User interaction may be required to open the popup but I'll check if it can be done automatically.

**Reply by sweepies** [reply]
Could we forego the pop up window in favor of a simple redirect? This is how most of my apps do it and it plays nicer with the provider initiating the login.

Maybe make it configurable?

**Reply by henrygd** [reply]
The native PocketBase oauth flow requires a popup. Is there a specific provider or browser that's not working properly for you?

**Reply by sweepies** [reply]
Gotcha. It works fine, just an extra click is required. Most apps can automatically redirect when oidc is the only configured sign in method.

I.e. launch -> a couple redirects -> user is signed in to the app on its landing page

versus 

launch -> Beszel sign in page -> click oidc login -> signed in
